Robot_Adapter/Read/Results/ResultUtils.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2018, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Collections.Generic;
+using System.Linq;
+using BH.oM.Base;
+using BH.oM.Structure.Results;
+using BH.oM.Structure.Requests;
+using BH.oM.Data.Requests;
+using RobotOM;
+using BH.oM.Common;
+using System;
+using System.Collections;
+
+namespace BH.Adapter.Robot
+{
+    public partial class RobotAdapter
+    {
+        /***************************************************/
+        /**** Protected Override Methods                ****/
+        /***************************************************/
+
+            
+
+        /***************************************************/
+        /**** Private Methods                           ****/
+        /***************************************************/
+
+        private double CheckGetValue(RobotResultRow row, int valuePosition)
```

Comment:
I'd prefer this to be called somehow more descriptively, like `TryGetValue` or `GetValueIfAvailable`. 
Also, what's our strategy to deal with not-found values? I'd generally prefer to see us always returning `null` out of methods like these if nothing is found.

Reply:
Strategy should be `null` unless otherwise appropriate (for valid error outputs) or constraints.

I prefer `TryGetValue` as that's consistent with our other methods of a similar nature and isn't so verbose as to become annoying :smile: